Christine/Lab2/part1/cpp_playground/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.1)
+project(
+    ccp_playground VERSION 1.0
+                  DESCRIPTION "Playing around with cmake"
+)
+
+#Added foo to play around with interfaces (bar implements foo)
+#I can delete if u want me to
+add_library(foo STATIC src/foo.cpp)
+add_library(bar STATIC src/bar.cpp)
+
+add_executable(target app/playing.cpp)
+
+target_include_directories(bar
+    PUBLIC 
+        ${PROJECT_SOURCE_DIR}/include
+)
+target_include_directories(foo
+    PUBLIC 
+        ${PROJECT_SOURCE_DIR}/include
+)
```

Comment:
You don't need to use the project source directory variable here, the path you give here to the include is already relative to this path